### ISSUE TITLE
Question - Pre-Built Shared Module included in EAR file.

### ISSUE THREAD

**Issue by Rojmanning** [TIBCOSoftware/bw6-plugin-maven#104]
Hi,

I am trying to work out how to include an already built shared module in an application during the maven build process.
I have a shared module version 1.0.0 built and secured in a repository, I would now like the build of my application to take this module from the repository and include it in the EAR file without rebuilding it.
It appears that if I include it as a module then it is built every time and will try to upload it back to the repository which break the strict version control.
But if I include it as a dependency in the parent pom the build is happy  and successful but the module is not included in the EAR file.

Any and all help greatly appreciated.

Thanks
Roger

**Comment by HiteshNembhwani** [comment]
I'm assuming the shared module is already added as a dependency to your application module. Is this correct? If yes, is your use-case to built an EAR with the updated shared module (i.e. jar in your use-case)

Try this with Maven 1.3.0 version though you'd still have to rebuild the EAR but can control the versioning via the pom files.

**Comment by awagle** [comment]
@Rojmanning closing this based on the comment from @HiteshNembhwani . Please reopen the issue if you still see this bug with 1.3.0

**Comment by Rojmanning** [comment]
Hi, thanks for the info.
I have installed the Maven 1.3.0 plug in but I can no longer build my workspace.
I have deleted and recreated the workspace but still get the following error.
-----
Errors occurred during the build.
Errors running builder 'BusinessWorks Project Validation Builder' on project 'sbs.bw.core.customer.interface'.
com/tibco/bw/design/external/dependencies/BWExternalDependenciesRegistry
-----
I have reinstalled the old plugin 1.2.2 and it works fine again. 
I have tried on BW633 and BW641
